.github/workflows/pin-cachix-release.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2022-2025 TII (SSRC) and the Ghaf contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: eval
+on:
+  push:
+    branches:
+      - main
```

Comment:
This is just for the WIP, right? I assume we don't want to pin all pushes to main?

Reply:
Why not? You suggest pin only stable tagged releases? Or may be use `ghaf-untrusted` repo for main

Reply:
There's only so much storage in our cachix cache. What happens when we pin more than we have storage?
We must have some process to delete the pins too, and remain somewhat selective as to what we choose to pin.